setup.py

```diff
@@ -41,7 +41,12 @@ def run_tests(self):
                   coverage, fail_under))
             raise SystemExit(1)
 
-VERSION = open('VERSION').read().strip()
+def _get_version():
```

Comment:
Oh, very interesting!  I just looked through my site-packages, and I see two sorts of things - directories like this, and single files.  So yes, you're totally right.